traits/tests/test_date.py

```diff
@@ -77,6 +86,7 @@ def test_assign_none(self):
         # allow_none=False in the trait definition does not work as expected.
         # (Ref: enthought/traits#495)
         obj = HasDateTraits(simple_date=UNIX_EPOCH)
+        self.assertIsNotNone(obj.simple_date)
```

Comment:
Slight strengthening of an existing test, for consistency with later tests. This doesn't relate to any change in behaviour.